hicommonwealth/commonwealth
# 6714 rendering unordered list

## Link to Issue
Closes: #6714 

## Description of Changes
- Modified the CSS to display all types of lists

## "How We Fixed It"

- Made the default list none 
- Explicitly set the checkbox style as a checkbox 

## Test Plan
- Go to create a thread or reply to a thread
- Copy the following:
```
- Test bullet1
- Test bullet 2
- Test bullet 3
- [ ]  Test checkbox 1
1. Test number 1
2. Test number 2
```
- Ensure the format is correct
